docs/contributing.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+First of all, thank you for considering to contribute. It means a lot to me! To contribute, make sure to 
+follow the steps given below.
+
+## Building
+
+To build ez_profile, you first need to fork this repository. Then you need to clone it using this command
+```sh
+git clone https://github.com/<your-username>/ez_profile.git
+```
+After this, enter the ez_profile folder:
+```sh
+cd ez_profile
+```
+If you want to contribute, you need to create a separate branch which would later be merged into `dev`.
+```sh
+git switch -c <branch-name>
+```
+Now make whatever changes you want to the source code. When you are done, build it by first installing 
+the developer dependencies:
+```sh
+pip install -r requirements.txt
+```
+Now install using this command:
+```sh
+pip install .
+```
+And to install in editable mode:
+```sh
+pip install -e .
+```
+
+When you are done, you can add and commit your changes and push then to GitHub.
+```sh
+git add . # adds the whole folder
```

Comment:
Don't encourage this. Someone might have random files they put there for testing purposes that they accidentally commit. Just tell them to add the files they intend to commit